src/js/components/Layer/__tests__/__snapshots__/Layer-test.js.snap

```diff
@@ -4032,7 +4032,7 @@ exports[`Layer position: bottom - full: false 1`] = `
   position: absolute;
   max-height: calc(100% - 0px - 0px);
   max-width: calc(100% - 0px - 0px);
-  border-radius: 4px;
+  border-radius: 4px 4px 0 0;
```

Comment:
Upon looking further into these, I don't think these snapshots should change since these tests are coming from using no theme (intelligentRounding should not be enabled) -- we should look into why it's suddenly changing.

Reply:
Oh I see below, you had accidentally left `intelligentRounding` enabled in base.js. Let's switch it back to undefined and these snapshot changes should go away

Reply:
oops okay let me verify that push it up now